app/controller/MapController.js

```diff
@@ -49,4 +50,5 @@ Ext.define('CpsiMapview.controller.MapController', {
             btn.getViewModel().get('polygonMeasureAreaTooltip');
         btn.tooltipStr = tTipStr;
     }
+
```

Comment:
This has nothing to do with this PR, or?

Reply:
Same here.